CoreScriptsRoot/Modules/Settings/Pages/Players.lua

```diff
@@ -117,12 +134,42 @@ local function Initialize()
 					if friendLabel and friendLabelText and friendLabelText.Text ~= '' then
```

Comment:
This might be a lot of work, I'd have no problem accepting the pull request without this and then we can do this later.